prototype_source/inductor_windows_cpu.rst

```diff
@@ -0,0 +1,127 @@
+How to use TorchInductor on Windows CPU
+=======================================
+
+**Author**: `Zhaoqiong Zheng <https://github.com/ZhaoqiongZ>`_, `Xu, Han <https://github.com/xuhancn>`_
+
+
+
+TorchInductor is a new compiler backend that compiles FX Graphs generated by TorchDynamo into optimized C++/Triton kernels.
```

Comment:
We should avoid using words like "new" as it gets outdated quickly:
Also, saying "compiler compiles" might be redundant, can we say "transform" instead?
```suggestion
TorchInductor is a compiler backend that transforms FX Graphs generated by TorchDynamo into highly optimized C++/Triton kernels.
```